crypto/kzg/src/trusted_setup.rs

```diff
@@ -21,6 +21,8 @@ struct G2Point([u8; BYTES_PER_G2_POINT]);
 /// See https://github.com/ethereum/consensus-specs/blob/dev/presets/mainnet/trusted_setups/trusted_setup_4096.json
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct TrustedSetup {
+    #[serde(rename = "g1_monomial")]
+    g1_monomial_points: Vec<G1Point>,
     #[serde(rename = "g1_lagrange")]
     g1_points: Vec<G1Point>,
```

Comment:
For consistency, we should rename `g1_points` to `g1_lagrange_points` and `g2_points` to `g2_monomial_points`.

Reply:
I see that was done in your other PR, but it should be here IMO.

<img width="459" alt="image" src="https://github.com/user-attachments/assets/e7367f39-c75d-49b3-a3fd-468efd266818">